app/flows/property_fire_safety_payment_flow/outcomes/payment_amount.erb

```diff
@@ -16,12 +16,42 @@
   <% end %>
 <% else %>
   <% text_for :title do %>
-    Thank you
+    You might have to pay <%= number_to_currency(calculator.leaseholder_costs, unit: "£", precision: 0) %> maximum
   <% end %>
 
   <% govspeak_for :body do %>
-    Leaseholder costs capped at <%= number_to_currency(calculator.leaseholder_costs, unit: "£", precision: 0) %>
+    You do not have to pay to replace cladding on your building.
 
-    Annual repayment capped at <%= number_to_currency(calculator.annual_leaseholder_costs, unit: "£", precision: 0) %>
+    The maximum you'll have to pay to fix other fire safety problems is <%= number_to_currency(calculator.leaseholder_costs, unit: "£", precision: 0) %>. The freeholder or 'landlord' (whoever is responsible for fixing problems with the building) can only charge you up to <%= number_to_currency(calculator.annual_leaseholder_costs, unit: "£", precision: 0) %> of this total in a year.
```

Comment:
Minor: we could store `number_to_currency(calculator.leaseholder_costs, unit: "£", precision: 0)` as a variable / helper method as its repeated.

Reply:
I've had the same thought - this will be coming in a future PR!